code/modules/goonchat/browserassets/css/browserOutput.css

```diff
@@ -398,7 +398,15 @@ h1.alert, h2.alert		{color: #000000;}
     0% { color: #0d0d0d; }
     25% { color: #410194; }
     50% { color: #7f17d8; }
-		75% { color: #410194; }
+	75% { color: #410194; }
+	100% { color: #3bb5d3; }
+}
+
+.phobia			{color: #dd0000;	font-weight: bold; animation: phobia 750ms infinite;}
+	@keyframes phobia {
+		0% { color: #0d0d0d; }
+		50% { color: #dd0000; }
+		100% { color: #0d0d0d; }
```

Comment:
The indentation here is wacky and there's some spurious line changes. Ditto below.